common/src/main/java/org/apache/falcon/security/SecurityUtil.java

```diff
@@ -115,6 +115,16 @@ public static boolean isAuthorizationEnabled() {
                 "falcon.security.authorization.enabled", "false"));
     }
 
+    /**
+     * Checks if CSRF filter is enabled in the configuration.
+     *
+     * @return true if falcon.security.csrf.enabled is enabled, false otherwise
+     */
+    public static boolean isCSRFFilterEnabled() {
```

Comment:
In this case, it is very possible that CSRF happens by intercepting the user name and making requests with it. This is one type of CSRF we want to prevent. And so yes, we need CSRF in this case.